heapoverflow/heapoverflow.json

```
@@ -0,0 +1,24 @@
+{
+    "key": "HeapOverflow",
+    "label": "HeapOverflow",
+    "display": "standalone",
+    "access": "external",
```

No ingress rules needed for this one, since it doesn't expose any ports:
```suggestion
    "access": "none",
```